ios/PurchasesHybridCommon/PurchasesHybridCommon/RCCommonFunctionality.h

```diff
@@ -69,21 +69,21 @@ signedDiscountTimestamp:(nullable NSString *)discountTimestamp
 signedDiscountTimestamp:(nullable NSString *)discountTimestamp
         completionBlock:(RCHybridResponseBlock)completion;
 
-+ (void)makeDeferredPurchase:(RCDeferredPromotionalPurchaseBlock)deferredPurchase
-             completionBlock:(RCHybridResponseBlock)completion;
+//+ (void)makeDeferredPurchase:(RCDeferredPromotionalPurchaseBlock)deferredPurchase
+//             completionBlock:(RCHybridResponseBlock)completion;
 
 + (void)setFinishTransactions:(BOOL)finishTransactions;
 
-+ (void)checkTrialOrIntroductoryPriceEligibility:(nonnull NSArray<NSString *> *)productIdentifiers
-                                 completionBlock:(RCReceiveIntroEligibilityBlock)completion;
+//+ (void)checkTrialOrIntroductoryPriceEligibility:(nonnull NSArray<NSString *> *)productIdentifiers
+//                                 completionBlock:(RCReceiveIntroEligibilityBlock)completion;
```

Comment:
These blocks didn't exist anymore and didn't seem to have a migration path in the docs 🤔 Need to look into this one some more